examples/run_full_prep.py

```diff
@@ -92,10 +92,11 @@
 
 # Fit prep
 prep_params = {
-    "ref_chs": ch_names_eeg,
-    "reref_chs": ch_names_eeg,
```

Comment:
ok, I will change it in the example but leave one with the 'eeg' and another with the channel list on the tests so that code coverage doesn't decline (as a result of the if statement selecting the case).

Reply:
Shouldn't we provide though the channel list way in an example too though?